riemann_client/client.py

```diff
@@ -6,7 +6,11 @@
 
 from __future__ import absolute_import
 
+import logging
 import socket
+from threading import RLock
+from threading import Timer
```

Comment:
I'm a little worried importing `threading` might break, since it's not available on on all platforms. Is there any possibility of doing this without threading?